ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/TransactionCommitterSpec.scala

```diff
@@ -19,6 +22,12 @@ import org.scalatest.{Matchers, WordSpec}
 
 class TransactionCommitterSpec extends WordSpec with Matchers with MockitoSugar {
   private val metrics = new Metrics(new MetricRegistry)
+  private val defaultConfigurationStateValueBuilder =
```

Comment:
It's preferred not to reuse builders as they are mutable:
```suggestion
  private def defaultConfigurationStateValueBuilder() =
```